.vale/styles/RedHat/Headings.yml

```diff
@@ -100,7 +100,7 @@ exceptions:
   - Jakarta
   - Java
   - Jave
-  - Jetbrains
+  - JetBrains
```

Comment:
Why do we need `Headings.yml` and `Spelling.yml` entries for JetBrains at all? Wouldn't the `CaseSensitiveTerms.yml` entry be enough?

Reply:
Without them, the test file produces the following complaints:
```
testfile.adoc
 1:1  warning  Use correct American English    RedHat.Spelling           
               spelling. Did you really mean                             
               'jetbrains'?                                              
 1:1  error    Use 'JetBrains' rather than     RedHat.CaseSensitiveTerms 
               'jetbrains'.                                              
 2:1  warning  Use correct American English    RedHat.Spelling           
               spelling. Did you really mean                             
               'Jetbrains'?                                              
 2:1  error    Use 'JetBrains' rather than     RedHat.CaseSensitiveTerms 
               'Jetbrains'.         
```

Reply:
Without them, the following test file
```
jetbrains
Jetbrains
JetBrains
= JetBrains
```
 produces the following complaints:
```
testfile.adoc
 1:1  warning  Use correct American English    RedHat.Spelling           
               spelling. Did you really mean                             
               'jetbrains'?                                              
 1:1  error    Use 'JetBrains' rather than     RedHat.CaseSensitiveTerms 
               'jetbrains'.                                              
 2:1  warning  Use correct American English    RedHat.Spelling           
               spelling. Did you really mean                             
               'Jetbrains'?                                              
 2:1  error    Use 'JetBrains' rather than     RedHat.CaseSensitiveTerms 
               'Jetbrains'.         
```

Reply:
I'm not sure what our "philosophy" on this is. 
- For the user experience, it makes sense to suppress the spelling warnings.
- For clarity and maintainability, it makes sense to reduce complexity and handle this using a single rule (CaseSensitiveTerms).

What are your thoughts on this?

Reply:
Ah OK yes, I stand corrected then. LGTM

Reply:
No correction intended, @aireilly. I had the same questions as you. It seemed strange to have a spelling and case-sensitive rules.